railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -25,6 +25,10 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 <%- end -%>
 <% if RUBY_ENGINE == 'ruby' -%>
+
+# Reduces boot times through caching; configured in config/boot.rb
+gem 'bootsnap', '~> 1.0.0', platforms: [:mri]
```

Comment:
Can we loosen this version constraint a bit?

Can bootsnap be installed on all platforms, even if it won't Do Cool Stuff?